Assets/Scripts/UI/SettingsUI.cs

```diff
@@ -61,12 +65,31 @@ void Awake()
         AddEndEditListener(ref _scaleSnappingText, value => ProjectData.scaleSnapStep = float.Parse(value));
 
         AddEndEditListener(ref _uiScaleText, value => CanvasManager.UiScale = float.Parse(value));
-
+        AddEndEditListener(ref _frameRateCap, value => PersistentData.FramerateCap = int.Parse(value));
         AddEndEditListener(ref _scriptLocationText, value => ProjectData.generateScriptLocation = value);
+        _framerateCapToggle.onValueChanged.AddListener(delegate { FrmerateCapToggleChanged(_framerateCapToggle); });
 
         CanvasManager.onUiChange.AddListener(SetDirty);
     }
 
+    private void FrmerateCapToggleChanged(Toggle framerateCapToggle)
```

Comment:
Die Funktion würde ich komplett auflösen. Das besprechen wir aber am besten in einem Call. Das ist sonst zu kompliziert